doc/telemetry.rst

```diff
@@ -97,81 +97,144 @@ Go Runtime metrics
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``tasks.nbWaiting``                   | This tracks the number of tasks waiting for being processed.             | number of       | gauge       |
-|                                       |                                                                          | tasks           |             |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``tasks.wait``                        | This measures the finally waited time for a task being processed.        | milliseconds    | timer       |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``task.<DepID>.<Type>.<FinalStatus>`` | This counts by deployment and task type the final status of a task.      | number of tasks | counter     |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
-| ``task.<DepID>.<Type>``               | This measures the task processing duration.                              | milliseconds    | timer       |
-+---------------------------------------+--------------------------------------------------------------------------+-----------------+-------------+
+~~~~~~~~~~~~~~~~~~~~~
+
+The **method** lablel represents the HTTP verb.
+The **path** lablel corresponds to the request URL where slashes are replaced by dashes.
+The **status** label represents a HTTP status codes (ie: 200, 404, 500, ...).
+
++------------------------+--------------------+------------------------------------------------------+--------------------+-------------+
+|       Metric Name      |        Labels      |              Description                             |        Unit        | Metric Type |
+|                        |                    |                                                      |                    |             |
++========================+====================+======================================================+====================+=============+
+| ``yorc.http.duration`` | method             | This measures the duration of an API call.           | milliseconds       | timer       |
+|                        | path               |                                                      |                    |             |
++------------------------+--------------------+------------------------------------------------------+--------------------+-------------+
+| ``yorc.http.total``    | method             |  This counts the number of API calls                 | number of requests | counter     |
```

Comment:
The sample output at the beginning of this page would need to be updated as it is still referencing metrics with the old naming scheme: yorc.http.<Method>.<Path>